.ci/test_r_package.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# set up R environment
+R_LIB_PATH=~/Rlib
+mkdir -p $R_LIB_PATH
+echo "R_LIBS=$R_LIB_PATH" > ${HOME}/.Renviron
+echo 'options(repos = "https://cran.rstudio.com")' > ${HOME}/.Rprofile
+export PATH="$R_LIB_PATH/R/bin:$PATH"
+
+# installing precompiled R for Ubuntu
+# https://cran.r-project.org/bin/linux/ubuntu/#installation
+# adding steps from https://stackoverflow.com/a/56378217/3986677 to get latest version
+if [[ $OS_NAME == "linux" ]]; then
+    sudo apt-get update
```

Comment:
I suspect `update` is not needed to be executed twice.

Reply:
hmm ok. I've tried removing it, let's see!